verifier.yml

```diff
@@ -129,11 +128,6 @@ components:
           $ref: "./components/EnvelopedVerifiablePresentation.yml#/components/schemas/EnvelopedVerifiablePresentation"
         options:
           $ref: "./components/VerifyPresentationOptions.yml#/components/schemas/VerifyPresentationOptions"
-    ProoflessVerifyPresentationRequest:
```

Comment:
It's important that we don't remove the ability to submit a VP w/o a proof that contains VCs that are to be verified. If we remove this, how do we signal that this is an acceptable use case?

Reply:
Added presentations that don't have proofs in 90a00a2246d983182dfcc4f0142d12681a38e275.